curvefs/src/client/inode_cache_manager.cpp

```diff
@@ -213,13 +184,16 @@ CURVEFS_ERROR InodeCacheManagerImpl::BatchGetInodeAttr(
         NameLockGuard lock(nameLock_, std::to_string(*iter));
         bool ok = iCache_->Get(*iter, &inodeWrapper);
         if (ok) {
-            InodeAttr tmpAttr;
-            inodeWrapper->GetInodeAttr(&tmpAttr);
-            attrs->emplace_back(tmpAttr);
-            iter = inodeIds->erase(iter);
-        } else {
-            ++iter;
+            if (!FLAGS_enableCto || OpenInodeCached(*iter) ||
+                inodeWrapper->IsDirty()) {
+                InodeAttr tmpAttr;
+                inodeWrapper->GetInodeAttr(&tmpAttr);
+                attrs->emplace_back(tmpAttr);
```

Comment:
`emplace_back(std::move(tmpAttr))`, maybe `push_back` is more clear

Reply:
fixed